### ISSUE TITLE
Accessibility:List: Name of button is not read, when highlighted button is accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3486]
Description:
Name of button is not read, when highlighted button is accessed. 
Read as “button to activate press spacebar”.
Also, tooltip “Title1” provided for the button does not convey meaningful information about the UI. 

Expected: Name of each & every UI should be conveyed when accessed. And also, meaningful text alternative defining the purpose of UI should be provided for each & every non-text UI element.

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94649243-dfdc0d80-0311-11eb-8270-8f5e58458864.png)

**Comment by sKudum** [comment]
# 3485, is conveys similar, hence closing this.
